Admin/AbstractAdmin.php

```diff
@@ -1983,6 +1983,14 @@ public function getPersistentParameters()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function sharePersistentParameters($code)
```

Comment:
@core23, no big deal, both of variants satisfy me. So, shall I change it and reopen the PR?